packages/experimental/MenuButton/src/MenuButton.types.ts

```diff
@@ -0,0 +1,41 @@
+import { ContextualMenuItemProps, ContextualMenuProps, SubmenuProps } from '@fluentui-react-native/contextual-menu';
+import { FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { ButtonProps } from '@fluentui-react-native/experimental-button';
+import { SvgProps } from 'react-native-svg';
+
+export const menuButtonName = 'MenuButton';
+
+export interface MenuButtonContext {
+  showContextualMenu?: boolean;
+}
+
+export interface MenuButtonState {
+  context: MenuButtonContext;
+}
+
+export interface MenuButtonTokens extends FontTokens, IForegroundColorTokens, IBackgroundColorTokens, IBorderTokens {}
```

Comment:
Been noticing this, but a suggestion that it may make styling/tokens easier to apply to the menu button in the future if MenuButtonTokens extended ButtonTokens, since it would make sense for a menu button to be as customizable as a button style-wise.